webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-student.js

```diff
@@ -137,7 +137,7 @@ export class SakaiRubricCriterionStudent extends RubricsElement {
           c.selectedRatingId = ed.selectedRatingId;
           if (ed.pointsAdjusted) {
             c.pointoverride = ed.points;
-            c.selectedvalue = selectedRatingItem.points;
+            c.selectedvalue = selectedRatingItem != null ? selectedRatingItem.points : 0; // Set selected value (points) to zero if no rating was selected
```

Comment:
Are you sure this will be null if not initialised above? I'd set it explicitly to null (selectedRatingItem = null), just to be sure. I'd also change that var to a let, just tighten the scope up a bit.

Reply:
I've pushed an update. Thanks for the review.